src/test/java/org/mastodon/mamut/feature/branch/exampleGraph/ExampleGraph4.java

```diff
@@ -0,0 +1,92 @@
+import org.mastodon.mamut.model.branch.ModelBranchGraph;
+
+/**
+ * Represents a {@link AbstractExampleGraph} with the following {@link ModelGraph} and {@link ModelBranchGraph}:
+ *
+ * <h1>Spot-Graph</h1>
+ * <pre>
+ *                                                Spot( 0, X=1,00, Y=2,00, Z=3,00, tp=0 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 1, X=0,00, Y=0,00, Z=0,00, tp=1 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 2, X=3,00, Y=6,00, Z=9,00, tp=2 )
+ *                       ┌───────────────────────────────────────────┴──────────────────────┐
+ *                       │                                                                  │
+ *  Spot( 11, X=12,00, Y=24,00, Z=36,00, tp=3 )                         Spot( 3, X=4,00, Y=8,00, Z=12,00, tp=3 )
+ *                       │                                                                  │
+ *                       │                                                                  │
+ *                       │                                              Spot( 4, X=5,00, Y=10,00, Z=15,00, tp=4 )
+ *                       │                                            ┌─────────────────────┴─────────────────────┐
+ *                       │                                            │                                           │
+ *  Spot( 13, X=14,00, Y=28,00, Z=42,00, tp=5 )   Spot( 8, X=9,00, Y=18,00, Z=27,00, tp=5 )   Spot( 5, X=6,00, Y=12,00, Z=18,00, tp=5 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                                                    │                       Spot( 6, X=0,00, Y=0,00, Z=0,00, tp=6 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                               Spot( 10, X=11,00, Y=22,00, Z=33,00, tp=7 )  Spot( 7, X=8,00, Y=16,00, Z=24,00, tp=7 )
+ * </pre>
+ * <h1>BranchSpot-Graph</h1>
+ * <pre>
+ *                        branchSpotA
+ * 	       ┌──────────────┴─────────────────┐
+ * 	       │                                │
+ * 	   branchSpotC                      branchSpotB
+ * 	                                 ┌──────┴───────┐
+ * 	                                 │              │
+ * 	                             branchSpotE branchSpotD
+ * </pre>
+ */
+public class ExampleGraph4 extends AbstractExampleGraph
```

Comment:
I merged ExampleGraph1, ExampleGraph3 and ExampleGraph4 together. I kept the ability of the resulting graph that it can be used also for a BranchSinuosityFeature test, since it does not infer with the exisiting tests in this repositoriy.